### ISSUE TITLE
Scrolling

### ISSUE THREAD

**Issue by jkatayama24** [kramis24/RiskProjectGroup4#7]
Would you be able to scan/scroll throughout the map to see everything?

**Comment by kramis24** [comment]
Yes, the map should be able to detect how much you move you finger for each touch event and should be able to recognise when the user is dragging on the map to know when to move it.
